src/citrine/informatics/executions/sample_design_space_execution.py

```diff
@@ -2,73 +2,37 @@
-):
+class SampleDesignSpaceExecution(Resource['SampleDesignSpaceExecution'], Execution):
     """The execution of a Sample Design Space task.
 
-    Possible statuses are INPROGRESS, SUCCEEDED, and FAILED.
-    Design executions also have a ``status_description`` field with more information.
+    Possible statuses are EXECUTING, SUCCEEDED, and FAILED.
+    Sample Design Space executions also have a ``status_detail`` field with more information.
+
     """
 
-    _paginator: Paginator = Paginator()
-    _collection_key = 'response'
-    _in_progress_statuses = ["INPROGRESS"]
-    _succeeded_statuses = ["SUCCEEDED"]
-    _failed_statuses = ["FAILED"]
-    _session: Optional[Session] = None
-    project_id: Optional[UUID] = None
     design_space_id: Optional[UUID] = None
 
-    uid: UUID = properties.UUID('id', serializable=False)
-    """:UUID: Unique identifier of the execution"""
-    status = properties.Optional(properties.String(), 'status', serializable=False)
-    """:Optional[str]: short description of the execution's status"""
-    status_description = properties.Optional(
-        properties.String(), 'status_description', serializable=False)
-    """:Optional[str]: more detailed description of the execution's status"""
-    status_info = properties.Optional(
-        properties.List(properties.String()),
-        'status_info',
-        serializable=False
-    )
-
-    """:Optional[List[str]]: human-readable explanations of the status"""
-    created_by = properties.Optional(properties.UUID, 'created_by', serializable=False)
-    """:Optional[UUID]: id of the user who created the resource"""
-    updated_by = properties.Optional(properties.UUID, 'updated_by', serializable=False)
-    """:Optional[UUID]: id of the user who most recently updated the resource,
-    if it has been updated"""
-    create_time = properties.Optional(properties.Datetime, 'create_time', serializable=False)
-    """:Optional[datetime]: date and time at which the resource was created"""
-    update_time = properties.Optional(properties.Datetime, 'update_time', serializable=False)
-    """:Optional[datetime]: date and time at which the resource was most recently updated,
-    if it has been updated"""
-
-    def __init__(self):
-        """The SampleDesignSpace executions are not directly instantiated by the user."""
-        pass  # pragma: no cover
-
-    def __str__(self):
-        return '<SampleDesignSpaceExecution {!r}>'.format(str(self.uid))
-
     def _path(self):
         return format_escaped_url(
             '/projects/{project_id}/design-spaces/{design_space_id}/sample/',
             project_id=self.project_id,
             design_space_id=self.design_space_id,
         )
 
+    @classmethod
+    def _pre_build(cls, data: dict) -> dict:
+        """Run data modification before building."""
+        status = data.get('status')
+        update_dict = {"status": status["minor"], "status_detail": status["detail"]}
```

Comment:
To maintain consistency, the `status` field should be the major status. Use `status_description` for the minor status.